packages/feedback-service/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM node:12-alpine
+FROM node:16.14.2-alpine
```

Just make it 16-alpine.
So that it can use the latest fixes and patches.
```suggestion
FROM node:16-alpine
```

---

Resolved